### DISCUSSION TITLE
When using passkey to login to Google, it seems that the set passkey cannot be found.

### DISCUSSION THREAD

**Discussion by likevi9528** [dani-garcia/vaultwarden#4038] [Q&A] [answered]
Browser: Chrome118.0.5993.120
vaultwarden: 1.30.0
bitwarden extension: 2023.10.1
OS: win 10 19043.1237

At the same time, I also used the official service and the bitwarden extension in the Edge browser but could not use the key to log in to Google.
Does the same situation happen to everyone?

**Comment by alexdelprete** [comment, marked as answer]
Confirmed. Tried 1.30 yesterday and tested passkey creation: it saves it, but when logging in with passkey, it says it can't find one.

**Reply by tessus** [reply]
The OP also mentioned that it doesn't work with Bitwarden either. So this basically means that it is most likely a client issue.

**Reply by alexdelprete** [reply]
yes I noticed, I just wanted to confirm it was happening not only to him.

I wonder if it's just with google: is there a passkey test site available?

**Reply by alexdelprete** [reply]
Tested with https://www.passkeys.io and https://webauthntest.identitystandards.io: it works beautifully.

So it's a specific issue with Google I guess.

**Reply by KSPSlice** [reply]
Can confirm the same. Using Microsoft Edge. Key has been set up but when I attempt to login, Bitwarden reports no passkey found. I then need to select use browser and another form of passkey I have set up.

**Comment by zvrh** [comment]
bitwarden issues：
https://github.com/bitwarden/clients/issues/6764

**Comment by alexdelprete** [comment]
Fixed in 2023.10.2: https://github.com/bitwarden/clients/releases/tag/browser-v2023.10.2
